apps/web/prisma/schema.prisma

```diff
@@ -797,3 +805,27 @@ model ResourceFeedback {
   @@index([resourceId])
   @@map("resource_feedback")
 }
+
+enum FeaturedBlockType {
+  base     
+  resource 
+  profile  
+
+  @@map("featured_block_type")
+}
+
+model FeaturedBlock {
```

Comment:
how do you know in which order to display them ? you need to store an order int  i think